src/app/file-manager/services/gcs-api.service.ts

```diff
@@ -92,11 +92,11 @@ export class GcsApiService extends GcsService {
     }
   }
 
-  public cancelExportsToGCP(): Observable<boolean> {
-    const items = this.getExportPendingFiles();
+  public cancelExportsToGCP() {
+    this.cancel = true;
```

Comment:
It looks like this is only used for GCP, if so, its name should reflect that.

Reply:
Changed to a more logic name